cypresse2e/cypress/fixtures/privacyPolicyContent.json

```diff
@@ -0,0 +1,206 @@
+      "firstStrTxt": "“Aggregated Data”",
+      "secondDivTypeTxt": "Aggregated Data may be derived from your Personal Data, but once in aggregated form it will not constitute Personal Data as this data does not directly or indirectly reveal your identity. However, if we combine or connect Aggregated Data with your Personal Data, We can directly or indirectly identify YOU. In this sense, we will treat the combined data as Personal Data which will be used in accordance with this Privacy Policy.",
+      "strongText": "Aggregated Data"
+    }
+  ],
+
+  "paraPolicyCxDiffTyp2": [
+    {
+      "id": 6,
+      "firstDivTypeTxt": "There are many occasions when YOU provide information that may enable Us to identify YOU personally while using the Services. The Personal Data we may collect from YOU is described in the following table:",
+      "firstHeadTxtP1": "Category of Personal Data",
+      "firstHeadTxtP2": "What it includes",
+      "firstBodyUTxtP1": "Identity Data",
+      "firstBodyTxtP2": "First name, surname, maiden name, last name, username, date of birth, gender, picture, signature, etc.",
+      "secondBodyUTxtP1": "Contact Data",
+      "secondBodyTxtP2": "Your home address, work address, email address and telephone numbers.",
+      "thirdBodyUTxtP1": "Professional Background",
+      "thirdBodyTxtP2": "Educational and professional history, interests and accomplishments and projects completed.",
+      "fourthBodyUTxtP1": "Online Presence Data",
+      "fourthBodyTxtP2": "Links to your public account pages at social media websites, links to personal websites, and other online materials related to YOU.",
+      "fifthBodyUTxtP1": "Content Data",
+      "fifthBodyTxtP2": "Any content YOU post to the Services not already included in another category, including without limitation, your profiles, questions, preference settings, answers, messages, comments and other contributions on the Services.",
+      "sixthBodyUTxtP1": "Communications Preferences",
+      "sixthBodyTxtP2": "Your preferences in receiving marketing from Us and our third parties and your communication preferences. If YOU correspond with Us by email or messaging through the Services, we may retain the content of such messages and our responses.",
+      "seventhBodyUTxtP1": "Behavioural Data",
+      "seventhBodyTxtP2": "Inferred or assumed information relating to your behaviour and interests, based on your online activity.",
+      "eigthBodyUTxtP1": "Technical Data",
+      "eigthBodyTxtP2": "Internet protocol (IP) address, your sign in data, browser type and version, time zone setting and location, browser plug-in types and versions, operating system and platform and other technology on the devices YOU use to access this website or use our services.",
+      "ninethBodyUTxtP1": "Health Data",
+      "ninethBodyTxtP2": "Health conditions from You, when accessing the questionnaire.",
+      "strongText": "What Personal Data We collect"
+    }
+  ]
+}
```

Comment:
I'm not sure if the paragraphs need to be separated by type of paragraph context. I think just ordering them by the IDs within "paraPolicyCx": [] for the main ones and "paraPolicyCxWithOrdList": [] for the listed ones are enough. What do you think?

Also, is paragraph number 8 is missing?

Reply:
The reason is because different paragraphs have different format, so i just group them having similar format in the same group.  I have a hard time to get the correct value when traversing through the DOM.  for paragraph 6 ,I see table involved and paragraph 8 I see paragraph ,list, then paragraph, I  move them separated to make it easy to handle the long function